gef-ui/server/controllers/amendments/helpers/map-submitted-to-checker-email-variables.ts

```diff
@@ -59,8 +61,15 @@ const mapSubmittedToCheckerEmailVariables = ({
     formattedFacilityValue = `${currencySymbol}${value}`;
   }
 
-  const recipientName = `${user.firstname} ${user.surname}`;
-  const sendToEmailAddress = user.email;
+  const makersName = `${user.firstname} ${user.surname}`;
+  const makersEmail = user.email;
+
+  const checkersName = `${checker.firstname} ${checker.surname}`;
+  const checkersEmail = checker.email;
+
+  const dateSubmittedByMaker = format(new Date(), DATE_FORMATS.DD_MMMM_YYYY);
+
+  const portalUrl = 'https://www.google.com';
```

Comment:
```suggestion
  const portalUrl = process.env.PORTAL_UI_URL;
```

You will need to add this in the docker-compose.yml as well as I belive this is only set for portal-api.

Reply:
done